src/test/tests/rendering/annot_macros.py

```diff
@@ -0,0 +1,186 @@
+# Print time with 20 width, leading sign, 10 after decimal pt.
+text.text = "Time=$time%+20.10f$"
+Test("printf_fmt_4")
+# Print time with 10 char width, 6 precision, exponential format.
+text.text = "Time=$time%+10.6e$"
+Test("printf_fmt_5")
+# Print time with %A (hex) to see full precision of actual value
+text.text = "Time=$time ($time%A$)"
+Test("printf_fmt_6")
+# Print first 3 chars of meshname
+text.text = "First 3 chars meshname='$meshname%.3s$'"
+Test("printf_fmt_7")
+# Print first 3 chars of meshname
+text.text = "Left justified in 6 chars meshname='$meshname%-6.6s$'"
+Test("printf_fmt_8")
+# Handling % and $ chars in string
+text.text = "State $index%02d$ of $numstates%02d$ = " + "%2d%%"%(100*float(13)/float(TimeSliderGetNStates()))
+Test("printf_fmt_9")
+text.text = "State $index is 18.3% into $numstates"
+Test("printf_fmt_10")
+text.text = "$cycle $US, $cycle$US"
+Test("printf_fmt_11")
+#text.Delete() just use the same annot object for below section
+
+TestSection("Text annotation file (tafile) macros")
+#
+# 71 float values to write to a tafile, value @ index 13 is 0.83457325
+#
+fvals = [1.00000000, 0.99899307, 0.99597429, 0.99094976, 0.98392959, 0.97492791, 0.96396286,
+    0.95105652, 0.93623487, 0.91952777, 0.90096887, 0.88059553, 0.85844879, 0.83457325, 0.80901699,
+    0.78183148, 0.75307147, 0.72279486, 0.69106265, 0.65793873, 0.62348980, 0.58778525, 0.55089698,
+    0.51289928, 0.47386866, 0.43388374, 0.39302503, 0.35137482, 0.30901699, 0.26603685, 0.22252093,
+    0.17855689, 0.13423327, 0.08963931, 0.04486483, 0.00000000, -0.04486483, -0.08963931, -0.13423327,
+   -0.17855689, -0.22252093, -0.26603685, -0.30901699, -0.35137482, -0.39302503, -0.43388374,
+   -0.47386866, -0.51289928, -0.55089698, -0.58778525, -0.62348980, -0.65793873, -0.69106265,
+   -0.72279486, -0.75307147, -0.78183148, -0.80901699, -0.83457325, -0.85844879, -0.88059553,
+   -0.90096887, -0.91952777, -0.93623487, -0.95105652, -0.96396286, -0.97492791, -0.98392959,
+   -0.99094976, -0.99597429, -0.99899307, -1.00000000]
+
+#
+# 71 string values to write to a tafile, value @ index 13 is "Monument"
+#
+svals = ["Ability", "Zealous", "Question", "Yarn", "Wisdom", "Venture", "Umbrella",
+    "Triumph", "Sincere", "Revelation", "Porcelain", "Optimism", "Novelty", "Monument",
+    "Latitude", "Kinship", "Journey", "Illumination", "Harvest", "Guidance", "Fidelity",
+    "Elixir", "Duration", "Curiosity", "Blueprint", "Aspire", "Artifact", "Validation",
+    "Unfold", "Theory", "Sanctuary", "Retreat", "Perspective", "Narrative", "Mastery",
+    "Longevity", "Keystone", "Intrigue", "Heritage", "Generation", "Fluid", "Enigma",
+    "Delight", "Craft", "Beacon", "Anonymous", "Achieve", "Whisper", "Vigilant", "Unity",
+    "Tranquil", "Symbolic", "Resilience", "Quest", "Purity", "Overture", "Mirth",
+    "Luminous", "Kindred", "Junction", "Insight", "Horizon", "Gradient", "Flourish",
+    "Essence", "Destiny", "Cycle", "Brevity", "Awakening", "Altruism", "Zenith"]
```

Comment:
I am confused; what are all of these for?

Reply:
Well, these are to test the `$TtafileX` (and friends) style macros. These macros work by storing lines of data in a `.txt` file, one for each timestate of the database. You can [read more about them](https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/using_visit/MakingItPretty/Annotations.html#named-database-values-in-text-annotations)

I had asked ChatGpt to create a random collection of 71 English language words to test `stafile1`. But, now that I think about it, those aren't all that useful in testing this feature. For example, you have to count in 13 states to find that the right string to have rendered after having called `TimeSliderSetState(13)` is `"Monument"`.

Should I adjust?

Reply:
I don't think it's a big deal; if you want to trim the list down a bit maybe that would be better. But if not no worries.